src/pages/home.js

```diff
@@ -327,17 +316,18 @@ export default function Home() {
               variant="primary"
               className="md:ml-2 ml-1"
               size="sm"
-              onClick={() => laTeXSepConvert('d2b')}
-            >
-              {t('dollar2bracket')}
-            </Button>
-            <Button
-              variant="primary"
-              className="md:ml-2 ml-1"
-              size="sm"
-              onClick={() => laTeXSepConvert('b2d')}
+              onClick={() => {
+                const newMode = displayConfig.latexDelimiter === 'dollar' ? 'bracket' : 'dollar';
+                setDisplayConfig({
```

Comment:
Could we consider refactoring the display config with the following enhancements?  
1. A dedicated hook.  
2. Robust APIs.  
3. For example, a new "setDisplayConfig" method to avoid redundant merges on the caller side, which also helps us hide the merging logic from the caller. This way, the caller may not need to be aware of the displayConfig.  
4. Another example is a better approach to avoid config access and magic strings, such as using "displayConfig.documentColor === 'bwwt'."